File: smp2017-android-muskan/cs50_psets/pset6/Muskan2108-cs50-2017-x-pset6/mario.py
import cs50
print("Height: ", end="")
n=cs50.get_int()
while True:
    if n>=0 and n<=23:
        break
    else:
        print("Height: ", end="")
        n=cs50.get_int()
for i in range(n):
    for j in range(i+2,n+1):
        print(" ",end="")
    for k in range(-1,i+1):
        print("#",end="")
    print("")
        

File: smp2017-android-aanchal/cs50  pset/pset1/AanchalHanda-cs50-2017-x-mario-less/mario.c
#include<stdio.h>
#include<cs50.h>
int main(void)
{
    printf("Height: ");
    int hei=get_int();
    int i,s,j;
  while(hei<0 || hei >23)
  { printf("Height: ");
      hei=get_int();}
    for(i=0;i<hei;i++)
    {
        for(s=hei-1;s>i;s--)
        printf(" ");
        for(j=0;j<i+2;j++)
        printf("#");
        printf("\n");
    }
}
    